### PR TITLE
doc: fix and improve links

### PR DESCRIPTION
**Pull request by rugk**
* the GitHub link is now locale-independent (auto-redirects)
* other links are broken due to moving files around apparently

### CODE REVIEW

File: res/documentation/creating new release.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ A new release is a good time to check if all dependencies are still up to date.
...
 - **Update presets version**. Check if `presetsVersion` and `nsiVersion` in the `build.gradle` still point to the latest version of the [iD presets](https://github.com/openstreetmap/id-tagging-schema/releases) and [name suggestion index](https://github.com/osmlab/name-suggestion-index/tags) respectively, check if the schema has any incompatible changes and update. 
-- **Update main contributor listing and other manual credit handling**. In general credits are updated automatically (see below), but [main contributors](app/src/main/res/raw/credits_main.yml) and list of ignored commits ([`skipCommit`](build.gradle.kts)) are handled manually.
+- **Update main contributor listing and other manual credit handling**. In general credits are updated automatically (see below), but [main contributors](../../app/src/main/res/raw/credits_main.yml) and list of ignored commits ([`skipCommit`](../../build.gradle.kts)) are handled manually.
```
Comment (westnordost) [inline res/documentation/creating new release.md:10]: y?

Comment (rugk) [reply res/documentation/creating new release.md:10]: links were 404 otherwise

Comment (westnordost) [reply res/documentation/creating new release.md:10]: is no absolute link possible?

Comment (rugk) [reply res/documentation/creating new release.md:10]: Sure, but that would also break when the repo is moved/renamed, whatever, so it always has disadvantages.

hmm https://docs.github.com/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#relative-links-and-image-paths-in-readme-files … let me try out sth.

Comment (rugk) [reply res/documentation/creating new release.md]: Though they recommend relative links there, I've changed it now. It works on GitHub.com fine at least.

**Pull request merged**